src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    protected int totalSalary;
+    protected int workingHours;
+    protected int salaryPerHour;
+    protected LocalDate workingDateFrom;
+    protected LocalDate workingDateTo;
+    protected LocalDate workingDay;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Names of constants should be capital letters
```suggestion
    private final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```